manifests/plugin/salt.pp

```diff
@@ -36,7 +36,7 @@
   Boolean $enabled = $foreman_proxy::plugin::salt::params::enabled,
   Foreman_proxy::ListenOn $listen_on = $foreman_proxy::plugin::salt::params::listen_on,
   String $user = $foreman_proxy::plugin::salt::params::user,
-  Optional[String[1]] $group = undef,
+  String $group = foreman-proxy,
```

Comment:
How is this any different? We already have `pick($group, $user)` and `$user` defaults to 'foreman-proxy'. Also, if you use strings, please don't use bare words. Technically they work, but they're frowned upon.
```suggestion
  String $group = 'foreman-proxy',
```

Reply:
The user is `root` here.

Reply:
Ah ok. Now I remember the situation. https://github.com/theforeman/puppet-foreman_proxy/pull/775 is the context. Salt ran as root, so it had permission. There's an installer parameter to change it. Note that back then we already suggested to use a different user/group.